File: timeish-app/src/app/models/activity.ts
export class Activity {
    id: number;
    date: Date;
    hours: number;
    description: string;
    pay: number;
    timeSheetId: number;

    constructor(id: number) {
        this.id = id;
    }

    // pay(hourlyPay: number): number {
    //     return hourlyPay * this.hours;
    // }
}

File: src/Tymish.Core/Entities/Activity.cs
using System;

namespace Tymish.Core.Entities
{
    public class Activity : BaseEntity
    {
        public DateTime Date { get; set; }
        public int Hours { get; set; }
        public string Description { get; set; }
        public decimal Pay { get; set; }
        public int TimeSheetId { get; set; }
    }
}